src/applications/caregivers/components/FormAlerts/SelectedFacilityInfoAlert.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FacilityAddress from '../FacilityAddress';
+
+const SelectedFacilityInfoAlert = ({ facility }) => {
+  if (!facility) return null;
+
+  return (
+    <va-alert status="info">
+      <h2 slot="headline" className="vads-u-font-size--h3">
```

Comment:
Are we sure this should be an H2? With its location on the page, it feels like it should be an H3. The breadcrumbs above its page location are an H2 and the next heading below with the actual page title is an H3.